makefiles/Makefile.cloud-docs

```diff
@@ -38,4 +38,4 @@ help:
 
 
 get-build-dependencies:
-	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/cloud-docs.yaml > ${REPO_DIR}/published-branches.yaml
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/{REPO_DIR}.yaml > ${REPO_DIR}/published-branches.yaml
```

Comment:
I don't know if we can use `REPO_DIR` as a part of the URL, or at least we need to slice it to get the part of the path containing the directory we want. When you run `pwd`, it gives you the full path to the directory:
![image](https://github.com/mongodb/docs-worker-pool/assets/20285369/f8b25004-e4ee-463e-b561-89c78945b0a4)